src/components/RewardWrapper/index.tsx

```diff
@@ -276,7 +311,7 @@ export function RewardWrapper({
 
       if (!isConnectedToG7) {
         throw new WarningError(
-          t('quest.noG7ConnectionSync.title', 'No G7 account linked'),
```

Comment:
we need the same string for all these errors. I checked and this doesn't seem to be shown to the player